CHANGELOG.md

```diff
@@ -120,6 +120,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Allow changing number of replicas of searchable snapshot index ([#11317](https://github.com/opensearch-project/OpenSearch/pull/11317))
 - Adding slf4j license header to LoggerMessageFormat.java ([#11069](https://github.com/opensearch-project/OpenSearch/pull/11069))
 - [BWC and API enforcement] Introduce checks for enforcing the API restrictions ([#11175](https://github.com/opensearch-project/OpenSearch/pull/11175))
+- Introduce cluster level setting to restrict replication type setting at index level ([#11583](https://github.com/opensearch-project/OpenSearch/pull/11583))
```

Comment:
should it be `cluster level setting` -> `cluster setting`? introducing cluster level setting at index level sounds confusing.